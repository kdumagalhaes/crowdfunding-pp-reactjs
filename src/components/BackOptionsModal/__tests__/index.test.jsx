import {render} from "@testing-library/react"
import {BackOptionsModal} from ".."

describe("Header", () => {
    it("initial component should match snapshot", () => {
        const {asFragment} = render(<BackOptionsModal />)
        expect(asFragment()).toMatchSnapshot()
    })
})