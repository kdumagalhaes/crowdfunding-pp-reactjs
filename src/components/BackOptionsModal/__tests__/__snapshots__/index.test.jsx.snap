// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header initial component should match snapshot 1`] = `<DocumentFragment />`;
